src/index.ts

```diff
@@ -1,12 +1,10 @@
 import * as angular from 'angular';
 
-import {FileInputDirective} from './input/FileInputDirective';
```

Comment:
This is removed but the FileInputDirective.ts still exists. On that note, it looks like references to SubmitFormDirective should also be removed.

Reply:
Thanks for catching this, fileinputdirective and submitformdirective have also been removed in that commit.